packages/app/navigation/bottom-tab-bar.tsx

```diff
@@ -16,23 +21,116 @@ import { BlurView } from "app/lib/blurview";
   const { width } = useWindowDimensions();
+  const isDarkMode = useIsDarkMode();
+  const isDark = dark !== undefined ? dark : isDarkMode;
+
+  const color = isDark ? colors.gray[100] : colors.gray[900];
+
+  const redirectToCreateDrop = useRedirectToCreateDrop();
+  return (
+    <View tw="flex-row bg-transparent pt-2">
+      {state.routes.map((route, index) => {
+        const { options } = descriptors[route.key];
+        const focused = state.index === index;
+
+        const onPress = () => {
+          const event = navigation.emit({
+            type: "tabPress",
+            target: route.key,
+            canPreventDefault: true,
+          });
+          Haptics.impactAsync();
+
+          if (route.name === "createTab") {
+            redirectToCreateDrop();
+            return;
+          }
+
+          if (!focused && !event.defaultPrevented) {
+            navigation.navigate({
+              name: route.name,
+              merge: true,
+              params: route.params,
+            });
+          }
+        };
+        const onLongPress = () => {
+          navigation.emit({
+            type: "tabLongPress",
+            target: route.key,
+          });
+        };
+
+        return (
+          <View key={route.key} tw="flex flex-1 items-center justify-center">
+            <Pressable tw="flex-1" onLongPress={onLongPress} onPress={onPress}>
+              {options.tabBarButton && (
+                <CreateTabBarIcon
+                  color={isDark ? "#000" : "#fff"}
+                  style={{ backgroundColor: isDark ? "#fff" : "#000" }}
+                />
+              )}
+              {options.tabBarIcon?.({ focused, color, size: 24 })}
+            </Pressable>
+          </View>
+        );
+      })}
+
+      <MotiView
+        style={{
+          position: "absolute",
+          top: 0,
+          height: 2,
+          backgroundColor: color,
+          width: width / state.routes.length,
+        }}
+        animate={{
+          translateX: (width / state.routes.length) * state.index,
+        }}
+        transition={{ type: "timing", duration: 250 }}
+      />
+    </View>
+  );
+};
+
+export const BottomTabbar = ({ state, ...rest }: BottomTabBarProps) => {
   const { isTabBarHidden } = useNavigationElements();
   const { isAuthenticated } = useUser();
   const { bottom: safeAreaBottom } = useSafeAreaInsets();
-  const isDark = useIsDarkMode();
-  const color = isDark ? colors.gray[100] : colors.gray[900];
-  const redirectToCreateDrop = useRedirectToCreateDrop();
   const nativeBottomTabBarHeightCallback = useBottomTabBarHeightCallback();
-
   const isHiddenBottomTabbar = !isAuthenticated || isTabBarHidden;
+  const isDark = useIsDarkMode();
+  const isHome = useDerivedValue(() => {
+    return state.index === 0 ? 1 : 0;
+  }, [state.index]);
+
+  const activeAnimatedStyle = useAnimatedStyle(() => {
+    return {
+      opacity: withTiming(isHome.value, { duration: 250 }),
+    };
+  }, [isHome.value]);
```

Comment:
If you want this to work on web, you need to pass `isHome` as dep instead of `.value`

Reply:
no, actully it's only used in native.